firewood/src/merkle/node.rs

```diff
@@ -842,3 +844,78 @@ impl Storable for Node {
         }
     }
 }
+
+#[cfg(test)]
+pub(super) mod tests {
```

Comment:
There are other ways to do this, but this was the fastest. A lot of our unit tests need cleanup anyway, so I hope this is fine for now.